BitwardenShared/UI/Auth/CompleteRegistration/MasterPasswordGuidance/MasterPasswordGuidanceView.swift

```diff
@@ -0,0 +1,132 @@
+import SwiftUI
+
+// MARK: - MasterPasswordGuidanceView
+
+/// A view that presents the user with guidance about how to create a strong master password.
+///
+struct MasterPasswordGuidanceView: View {
+    // MARK: Properties
+
+    /// The `Store` for this view.
+    @ObservedObject var store: Store<Void, MasterPasswordGuidanceAction, Void>
+
+    // MARK: View
+
+    var body: some View {
+        VStack(spacing: 16) {
+            VStack(spacing: 0) {
+                instructionsView
+
+                detailedInstructionsView
+            }
+            .background(Asset.Colors.backgroundTertiary.swiftUIColor)
+            .clipShape(RoundedRectangle(cornerRadius: 10))
+
+            passwordGeneratorButton
+        }
+        .scrollView()
+        .navigationBar(title: Localizations.masterPasswordHelp, titleDisplayMode: .inline)
+        .toolbar {
+            cancelToolbarItem {
+                store.send(.dismiss)
+            }
+        }
+    }
+
+    // MARK: Private views
+
+    /// The main instructions.
+    private var instructionsView: some View {
+        VStack(spacing: 0) {
+            VStack(alignment: .leading, spacing: 8) {
+                Text(Localizations.whatMakesAPasswordStrong)
+                    .styleGuide(.title3, weight: .semibold)
+
+                Text(Localizations.strongPasswordDescriptionLong)
+                    .styleGuide(.subheadline)
+            }
+            .frame(maxWidth: .infinity, alignment: .leading)
+            .foregroundStyle(Asset.Colors.textPrimary.swiftUIColor)
+            .padding(.all, 24)
```

Comment:
⛏️ No need for `.all` as it's the default.
```suggestion
            .padding(24)
```